plugins/katello/3.3/release_notes/release_notes.md

```diff
@@ -4,16 +4,26 @@ title: Release Notes
 version: 3.3
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.3 (Baltic Porter) Release Notes
 
-{% if page.version and page.version != '3.3' %}
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ page.version }}/CHANGELOG.md).
-{% else %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
-{% endif %}
 
 ## Features
 
+### Updated UI
+
+### Improved Manifest Import and Delete
+
+### Username and Password for Repositories
+
+### Content View Version Promotion to Multiple Environments
```

Comment:
You might group this one and the next with "Content View Improvements" and add:

* Arch available for content view package filters
* Allow Composite content views to be publish containing components with different copies of the same repositories, combining the contents as part of the composite.